### DISCUSSION TITLE
I want to get the sales data using Tiktok API.

### DISCUSSION THREAD

**Discussion by mono300genuine** [EcomPHP/tiktokshop-php#44] [Q&A] [answered]
Hello
I am new here in tiktok developer guide.
just to see that you could track the number of sales a product gets.
I want to get the sales data from tiktok shop using tictok API.
In short, I want to make tiktok shop sales tracking tool.
The goal is to determine if TikTok provides endpoints that expose product sales data.
Is it possible to get the sales product data using tiktok API?

I would like to establish a long-term partnership with someone who actually helps me.

**Comment by nVuln** [comment]
you can use Order API to download data of all your orders
then run your own report tool to generate whatever you want

**Comment by mono300genuine** [comment]
@nVuln Thx
this is third party api?
could you provide refer link I can use?

**Reply by nVuln** [reply]
official API provded by Tiktok, more detail here: https://partner.tiktokshop.com/docv2/page/650b1b4bbace3e02b76d1011

**Reply by mono300genuine** [reply]
> official API provded by Tiktok, more detail here: https://partner.tiktokshop.com/docv2/page/650b1b4bbace3e02b76d1011

Can't find another API endpoint that can retrieve sales data for TikTok Shop's products?
What I mean is, have you seen any other useful endpoints?

**Reply by nVuln** [reply]
you can download data of your order by using above API, write a custom report on your system and you can know how many sales each product

**Reply by mono300genuine** [reply]
I mean third party API.

**Reply by nVuln** [reply]
no I don't know

**Reply by mono300genuine** [reply, marked as answer]
Thank you, @nVuln 
could you leave you gmail?

**Comment by mono300genuine** [comment]
okay, thank you.
